packages/core/database/src/schema/schema.ts

```diff
@@ -34,7 +34,7 @@ const createColumn = (name: string, attribute: Attribute): Column => {
   };
 };
 
-const createTable = (meta: Meta): Table => {
+const createTable = (meta: Meta, options: MetadataOptions): Table => {
```

Comment:
To avoid exposing  & passing arround the metadata options could you store them in the metadata and expose a meta.getOptions() fn instead to keep a bit of encapsulation

Reply:
Or it's a matter of using another type :)